src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -113,7 +113,12 @@
 CC_Provider_Namespace_Not_Registered_Fault_Type = "Connected Cluster Provider MS.K8 namespace not registered"
 Default_Namespace_Does_Not_Exist_Fault_Type = "The default namespace defined in the kubeconfig doesn't exist on the kubernetes cluster."
 ClusterConnect_Not_Present_Fault_Type = "cluster-connect-feature-unavailable"
-CLIENT_PROXY_VERSION = '1.3.018101'
+KAP_1P_Server_App_Scope = "6256c85f-0aad-4d50-b960-e6e9b21efe35/.default"
+KAP_1P_Server_AppId = "6256c85f-0aad-4d50-b960-e6e9b21efe35"
+Get_PublicKey_Info_Fault_Type = 'Error while fetching the PoP publickey information from CSP'
```

Comment:
client proxy?

Reply:
Since we will be doing the GET call on CSP's endpoint - "https://localhost:{api_server_port}/identity/poppublickey to get the publickey" to get the publickey, I mentioned CSP in the fault type also

Reply:
just realized that you meant to rename CSP to client proxy

Reply:
Done. renamed it to client proxy

Reply:
CSP is still present in error message. Have you pushed the change?